etc/taskcluster/decision_task.py

```diff
@@ -37,8 +37,8 @@ def tasks(task_for):
             windows_arm64,
             windows_uwp_x64,
             macos_unit,
-            linux_wpt,
-            linux_wpt_layout_2020,
+            # linux_wpt,
+            # linux_wpt_layout_2020,
```

Comment:
We should make the `test-wpt` branch just below this an empty list, too.

Reply:
Oh, I see - for simplicity's sake, should disable the macos_wpt tests as well, and empty the try-wpt and and try-wpt-mac branches below as well.